src/Form/Layout.php

```diff
@@ -209,8 +210,10 @@ public function recursiveRender()
             $template->trySet('label_for', $el->id . '_input');
             $template->set('control_class', $el->getControlClass());
 
-            // backward compatibility - will be removed in dec-2020
-            $template->trySet('field_class', $el->getControlClass());
+            // BC-break exception
+            if ($template->hasTag('field_class')) {
+                throw new Exception('field_class region has be deprecated. Use control_class instead');
```

Comment:
the same for `AboveFields` when used

Reply:
the problem is AboveFields is set directly in dev custom code. I kept the region in the form template for BC

Reply:
But we can check if set and throw. There should be not that many usages and search/replace is quick...

Reply:
For this I need to remove the region in the template (I kept it for now)

Reply:
If absolutely needed, I this we should map the old region in render in PHP and use only the new one in template.

Reply:
done now